controllers/ppo/ppo.py

```diff
@@ -71,41 +59,21 @@ def compute_control(
         done: bool | None = None,
         info: dict | None = None,
     ) -> tuple[Command, list]:
-        """Pick command sent to the quadrotor through a Crazyswarm/Crazyradio-like interface.
-
-        INSTRUCTIONS:
-            Re-implement this method to return the target position, velocity, acceleration,
-            attitude, and attitude rates to be sent from Crazyswarm to the Crazyflie using, e.g., a
-            `cmdFullState` call.
-
-        Args:
-            ep_time: Episode's elapsed time, in seconds.
-            obs: The environment's observation [drone_xyz_yaw, gates_xyz_yaw, gates_in_range,
-                obstacles_xyz, obstacles_in_range, gate_id].
-            reward: The reward signal.
-            done: Wether the episode has terminated.
-            info: Current step information as a dictionary with keys 'constraint_violation',
-                'current_target_gate_pos', etc.
-
-        Returns:
-            The command type and arguments to be sent to the quadrotor. See `Command`.
-        """
-        action, next_predicted_state = self.model.predict(obs, deterministic=True)
-        action = transform_action(action, drone_pos=obs[:3])
-
-        # Prepare the command to be sent to the quadrotor.
-        command_type = Command.FULLSTATE
-        x = float(action[0])
-        y = float(action[1])
-        z = float(action[2])
-        target_pos = np.array([x, y, z])
-        target_vel = np.zeros(3)
-        target_acc = np.zeros(3)
-        target_yaw = float(action[3])
-        target_rpy_rates = np.zeros(3)
-        args = [target_pos, target_vel, target_acc, target_yaw, target_rpy_rates, ep_time]
-
-        # logger.info(f"Predicted action: {action}")
-        # logger.info(f"Args: {args}")
-        return command_type, args
-
+        """Pick command sent to the quadrotor through a Crazyswarm/Crazyradio-like interface."""
+        if ep_time - 2 > 0 and info["current_gate_id"] != -1:
```

Comment:
shouldnt it here use the information of the state machine something like `if self.state_machine.state == DroneState.PolicyControl: <then do policy control>`

Reply:
moved logic to deploy